x-pack/plugins/maps/public/classes/layers/new_vector_layer_wizard/wizard.tsx

```diff
@@ -124,16 +148,26 @@ export class NewVectorLayerEditor extends Component<RenderWizardArguments, State
   };
 
   render() {
-    if (this.state.createIndexError) {
+    if (!this.state.userHasIndexWritePermissions) {
```

Comment:
What about errors where createIndexError but not userHasIndexWritePermissions? These cases still need to diaplay error messages.

Reply:
The only other type of errors handled here are index name errors, these are still passed through to the index name form [here](http://github.com/aaronjcaldwell/kibana/blob/bd467d4a627372bfd0b81949847942226723d7aa/x-pack/plugins/maps/public/classes/layers/new_vector_layer_wizard/wizard.tsx#L181-L181).

Reply:
Actually scratch that. There is something wonky here that either needs clarifying or revision